### PR TITLE
PSR-1070 | View Only Task List

### PR DESCRIPTION
**Pull request by ccatosdevelopment**

### CODE REVIEW

File: app/controllers/nonsipp/memberdetails/upload/FileUploadErrorSummaryController.scala [hunk 1]
```diff
@@ -78,7 +78,7 @@ object FileUploadErrorSummaryController {
       heading = "fileUploadErrorSummary.heading",
       description = Some(
         ParagraphMessage("fileUploadErrorSummary.paragraph") ++
-          Heading2("fileUploadErrorSummary.heading2", LabelSize.Medium) ++
+          Heading2("fileUploadErrorSummary.heading2", HeadingSize.Medium) ++
```
Comment (kberec) [inline app/controllers/nonsipp/memberdetails/upload/FileUploadErrorSummaryController.scala:81]: These heading size related changes don't seem related to the tasklist, is this fixing some existing issue? Or just making the code more flexible?

**Review by kberec — APPROVED** [review]

File: app/controllers/nonsipp/memberdetails/upload/FileUploadErrorSummaryController.scala (hunk 1 above)
Comment (ccatosdevelopment) [reply app/controllers/nonsipp/memberdetails/upload/FileUploadErrorSummaryController.scala:81]: This was some of the custom CSS that broke after the library update, so I fixed it as part of that same commit.
